### ISSUE TITLE
Support to attach files to Issue

### ISSUE THREAD

**Issue by laysauchoa** [dblock/create-a-github-issue#9]
Greetings from the OpenSearchCon 👋 

Thanks for forking this repo and providing maintenance for this action. I've a question about whether would be possible to support attaching files to the issue? Possible files created in steps before running this action. 

I'd like to be able to attach files to the issue. GitHub issues have limitation of characters and saving to the environment variable comes with limitations of size as well. What do you think?

**Comment by dblock** [comment]
Certainly. I found https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files talking about "attaching files", but I don't know what the API looks like to support that.
